Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/Parse/Inlines/ImageInline.cs

```diff
@@ -117,6 +117,7 @@ internal static Common.InlineParseResult Parse(string markdown, int start, int e
             }
 
             string url = TextRunInline.ResolveEscapeSequences(markdown, urlStart, pos);
+            url = url.StartsWith("/") ? string.Format("ms-appx://{0}", url) : url;
```

Comment:
Should we have a property which lets the developer decide where the base url for relative links should be (and then default to `ms-appx://`)?

This would let them specify an alternate assets folder (`ms-appx://Assets`) or an actual url on the web (`http://www.mysite.com/Images`) as a baseline instead.

Reply:
@michael-hawker if the URL starts with `http://` it renders by default. that is the reason why url checks for `/` only. IDK how much of a help a separate property will be. Also the LinkClicked events does not apply on Images. It renders if it can or else it wont.

@nmetulev thoughts?

Reply:
I think @michael-hawker is referring to `string.Format("ms-appx://{0}", url)`, and changing it to somthing like `string.Format("{0}://{1}", prefix, url)` where prefix can be set by the developer to anything and defaults to ms-appx:/// if not set

Reply:
Exactly, I was thinking the full prefix string.Format("{0}{1}", prefix, url) where prefix default would be "msappx://"

Reply:
Thanks for fixing my example :)

Reply:
Got it. Will add new Property and update commit.